geth/api/backend_test.go

```diff
@@ -14,9 +14,27 @@ import (
 	"github.com/status-im/status-go/geth/node"
 	"github.com/status-im/status-go/geth/params"
 	. "github.com/status-im/status-go/geth/testing"
+	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 )
 
+func TestNodeStartWithUpstream(t *testing.T) {
```

Comment:
I created it as a separate test as some of the `BackendTestSuite` tests time out and it's not possible to run just a single one.